meinberlin/apps/bplan/serializers.py

```diff
@@ -42,6 +43,7 @@ class BplanSerializer(serializers.ModelSerializer):
         required=False,
         write_only=True,
     )
+    image_alt_text = ImageAltTextField(image_name=_("Tile image"))
     image_copyright = serializers.CharField(
         required=False,
         write_only=True,
```

Comment:
why do we not use the copyright field from a4 here?

Reply:
Maybe it's different in the serializer?